.changeset/pink-readers-marry.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Is this needed?

Reply:
Yes, every PR in this repo needs a changeset :)